### PR TITLE
Implement basic api tests

### LINKED ISSUE #39: Implement tests frameworks
I think it's better to create not only unit tests, but also automation one.This project should have test framework (or frameworks), as we have already discussed. 

I need to ask: do we need UI-testing for this project (I suppose you might know "Selenium")? Or we just use some functional tests to check flask? 

Also I'm going to find better framework to create unit tests (testtols, py.test etc). Please, if you have any ideas about this, write it here.

### PR DESCRIPTION
**Pull request by zemuvier**
close #39

### CODE REVIEW

**Comment by zemuvier** [comment]
If we want to run tests in travis, we need to configure installing of our project in travis too. For now I won't run these tests on CI, but I think we really need it

**Comment by rkhozinov** [comment]
You already have a docker implementation, so you could deploy your app in container using travis and then run tests. It’s really big movement for your project.
@nikitabelonogov please help with deploy

**Comment by rkhozinov** [comment]
Recently I’ve found the [pyresttest](https://github.com/svanoort/pyresttest) library. May be it could help you to get more readable tests and simple integration test flow between the team.

File: app/routs.py [hunk 1]
```diff
@@ -10,5 +10,5 @@ def index():
 
 
 @app.errorhandler(404)
-def page_not_found(e):
+def page_not_found():
```
Comment (nikitabelonogov) [inline app/routs.py]: it doesn't work like that.

**Comment by gaassassins** [comment]
@zemuvier please, can you write something like "how to run test" in old branch with README?

**Pull request merged**